msal/src/Microsoft.Identity.Client/TokenCache.cs

```diff
@@ -345,13 +365,25 @@ private MsalAccessTokenCacheItem FindAccessTokenCommon
                 }
 
                 requestParams.RequestContext.Logger.Info("Matching entry count -" + tokenCacheItems.Count);
-
-                IEnumerable<MsalAccessTokenCacheItem> filteredItems =
-                    tokenCacheItems.Where(
-                            item =>
-                                item.ScopeSet.ScopeContains(requestParams.Scope));
-
-                requestParams.RequestContext.Logger.Info("Matching entry count after filtering by scopes - " + filteredItems.Count());
```

Comment:
> requestParams.RequestContext.Logger.Info("Matching entry count after filtering by scopes - " + filteredItems.Count()); [](start = 15, length = 119)

what's the reason to remove the log statement?

Reply:
Accidnet